core/dbt/cli/params.py

```diff
@@ -518,6 +524,22 @@
     default=(),
 )
 
+sample = click.option(
+    "--sample/--no-sample",
+    envvar="DBT_SAMPLE",
+    help="Run in sample mode, creating only samples of models where possible",
+    default=False,
+    is_flag=True,
```

Comment:
Hide, and later unhide. I.e. `hidden=True`

Reply:
For clarification, we want to hide these while we're in alpha / gated behind an environment variable. Once we remove the environment variable, we'll unhide these